src/moveit_visual_tools.cpp

```diff
@@ -35,10 +35,10 @@
 // Author: Dave Coleman
 // Desc:   Simple tools for showing parts of a robot in Rviz, such as the gripper or arm
 
-#include <moveit_visual_tools/moveit_visual_tools.h>
+#include <moveit_visual_tools/moveit_visual_tools.hpp>
```

Comment:
hmm.. we haven't decided to switch to `.hpp` for MoveIt, yet. At some point we might want to, but for now we should keep it consistent and stick to `.h`.